package/gluon-ebtables-source-filter/check_site.lua

```diff
@@ -0,0 +1,2 @@
+need_string_match('prefix4', '^%d+.%d+.%d+.%d+/%d+$', false)
+need_string_array_match('additional_prefix6', '^[%x:]+/%d+$', false)
```

Comment:
What would you guys think about "extra_prefix6" or "extra_prefixes6"? (so similar to EXTRA_CFLAGS/LDFLAGS, for instance)